gulpfile.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+var gulp = require('gulp');
+var nodemon = require('gulp-nodemon');
+var ngrok = require('ngrok');
+
+// Environment variables to be set for bingo API
+var env = {
+  DB_HOST : 'localhost',
+  DB_PORT : '3306',
+  LOG_DIR : 'logs',
+  JWT_SECRET : 'thambi-ithu-booth-pa',
```

Comment:
we shouldn't be exposing any private tokens on public forums. The way we do it is 

- have config.json in local
- add config.json to .gitignore ( so that secrets doesn't get added to source control and it lies local to your machine )
- var config require(config.json) 
- use config.jwtSecret

and moreover , we don't need JWT_SECRET here, so please remove it.